config/packages/security.yaml

```diff
@@ -1,22 +1,15 @@
 security:
+    enable_authenticator_manager: true
```

Comment:
Dit hoeft dacht ik niet meer vanaf 6.4 en is deprecated

Reply:
Good point, removed that.